imhotep/tools.py

```diff
@@ -53,7 +53,7 @@ def invoke(self, dirname, filenames=set(), linter_configs=set()):
             to_find = ' -o '.join(['-name "*%s"' % ext
                                    for ext in self.get_file_extensions()])
 
-        cmd = 'find %s %s | xargs %s' % (
+        cmd = 'find %s -path "*/%s" | xargs %s' % (
```

Comment:
this is making invalid find commands...

Reply:
@gordol sorry for the late reply, I'm lacking some context here but looking to merge this if I can. Can you remember what was invalid about the commands it's generating here?